src/interfaces/Todo.ts

```diff
@@ -0,0 +1,16 @@
+import { User } from './User';
+
+export interface UnpreparedTodo {
+  id: number,
+  title: string,
+  completed: boolean,
+  userId: number,
+}
+
+export interface PreparedTodo {
+  id: number,
+  title: string,
+  completed: boolean,
+  userId: number,
+  user: User | null,
+}
```

Comment:
Having two interfaces in such way does not make much sense
You can merge them and have just one
```suggestion
export interface Todo {
  id: number,
  title: string,
  completed: boolean,
  userId: number,
  user?: User // notice - the user is optional
}
```